utilities/src/parallel.rs

```diff
@@ -32,26 +32,26 @@ impl<'a, T> ExecutionPool<'a, T> {
         Self { jobs: Vec::with_capacity(cap) }
     }
 
-    #[cfg(feature = "parallel")]
+    #[cfg(not(feature = "serial"))]
     pub fn add_job<F: 'a + FnOnce() -> T + Send>(&mut self, f: F) {
         self.jobs.push(Box::new(f));
     }
 
-    #[cfg(not(feature = "parallel"))]
-    pub fn add_job<F: 'a + FnOnce() -> T>(&mut self, f: F) {
+    #[cfg(feature = "serial")]
+    pub fn add_job<F: 'a + FnOnce() -> T + Send>(&mut self, f: F) {
```

Comment:
Why add the send marker trait? Not against it, just curious about the motivation.